src/client/methods.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
Maybe rename `methods.[c|h]` to `method-help` (and move the `method_version` to `main.c` since its really small) to be consistent with the others. 
Based on the file name I first assumed it contains the definition of the `Method` struct, but its actually located in `libbluechi/cli/command.h`.

Reply:
Done. You are right, this way is better. I wanted for `methods.h` to hold all includes for all `method-*.h` but really this is cleaner